about.html

```diff
@@ -0,0 +1,135 @@
+        </div>
+        <p class="about-description">
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+            dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+            book.
+        </p>
+        <p class="about-contact">Please contact us per email for any further questions about the website</p>
+        <address class="about-address"><a class="about-email-a" href="mailto:contact@poeticlyrics.com">contact@poeticlyrics.com</a></address>
+    </section>
+    <section class="about-logo">
+        <h2 class="about-h2">
+            Poetic Song Lyrics logo
+        </h2>
+        <p class="logo-desc">
+            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+            dummy text ever since the 1500s.
+        </p>
+        <img src="./images/high-res-logo.png" class="about-logo-img" alt="High res version of logo">
+    </section>
+    <section class="about-past">
+        <h2 class="past-title">
+            See the history of our Project
+        </h2>
+        <p class="past-description">
+            Take a look at our evolution throught the years since our early beginnings.
+        </p>
+        <div class="past-container">
+            <img src="./images/ccsummit2013.png" class="past-imgs" alt="project's own .com domain, in 2013">
+            <img src="./images/ccsummit2011.png" class="past-imgs" alt="project's first great reform, in 2011">
+        </div>
+    </section>
+    <section class="about-partners">
+        <p class="partners-title">
+            Partner
+        </p>
+        <div class="partners-above">
+            <img src="./images/partners/mozilla.png" class="partner-logo" alt=" logo">
+            <img src="./images/partners/Google.png" class="partner-logo" alt=" logo">
+            <img src="./images/partners/navar.png" class="partner-logo" alt=" logo">
+        </div>
+        <div class="partners-under">
+            <img src="./images/partners/daumkakao.svg" class="partner-logo" alt=" logo">
+            <img src="./images/partners/airbnb.png" class="partner-logo" alt=" logo">
+        </div>
+    </section>
+    <section class="about-footer">
+        <div class="footer-left">
+            <img src="./images/icons/cc.srr.svg" alt="Creative commons logo">
+        </div>
+        <div class="footer-right">
+            <p class="footer-disclaimer-about">
+                2022 Creative Commons.<br>Some Rights Reserved.
+            </p>
+            <p class="footer-dsk">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+                dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+                book.
+            </p>
```

Comment:
- [ ] Let's space things out between the partner's section and the footer by giving them proper `margin` and `padding` so that your page looks professional and also to improve the UI. 
- [ ] Also let's give the footer some padding so that it does not stick to the page. 


Your Implementation | Cindy's Design
:---:|:---:
![](https://i.imgur.com/URZGGfv.png) | ![](https://i.imgur.com/I1OjmSN.png)